Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/AnomalyDescriptor.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.randomcutforest.parkservices;
+
+import static com.amazon.randomcutforest.CommonUtils.checkArgument;
+
+import java.util.Arrays;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import com.amazon.randomcutforest.returntypes.DiVector;
+
+@Getter
+@Setter
+public class AnomalyDescriptor {
+
+    public static int NUMBER_OF_EXPECTED_VALUES = 1;
+
+    // anomalies should have score for postprocessing
+    double rcfScore;
+    // the following describes the grade of the anomaly in the range [0:1] where
+    // 0 is not an anomaly
+    double anomalyGrade;
+
+    // same for attribution; this is basic RCF attribution which has high/low
+    // information
+    DiVector attribution;
+
+    // timestamp (basically a sequence index); kept as long for potential future use
+    long timeStamp;
+
+    // confidence, for both anomalies/non-anomalies
+    double confidence;
+
+    // number of trees in the forest
+    int forestSize;
```

Comment:
removed.

Reply:
But on second thoughts - we are not persisting these values, would it hurt if we had this value. If we have to compare scores from different size forests (without access to the model) -- say after the fact, this can be useful.